src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: auto auto 40px;
+  }
+
+  &__title {
+    color: $color-main;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__count {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-main;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__description {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $color-main;
```

Comment:
The variable `$color-main` is used here but not defined in this file. Ensure it is defined in your SCSS setup.